src/sasquatchbackpack/cli.py

```diff
@@ -180,10 +183,30 @@ def usgs_earthquake_data(
             click.echo(result)
         click.echo("------")
     else:
-        click.secho("SUCCESS", fg="orange")
+        click.secho("SUCCESS! (kinda)", fg="yellow")
         click.echo("------")
         click.echo("No results found for the provided criteria :(")
         click.echo("------")
+        return
+
+    click.confirm("Do you want to post the above data?", abort=True)
+
+    click.echo("Sending data...")
+
+    config = sources.USGSConfig(
+        total_duration, radius, coords, magnitude_bounds
+    )
+    source = sources.USGSSource(config)
+
+    backpack_dispatcher = sasquatch.BackpackDispatcher(
+        source, sasquatch.DispatcherConfig()
+    )
+    result = backpack_dispatcher.post()
+
+    if "Error" in result:
+        click.secho(result, fg="red")
+    else:
+        click.secho("Data successfully sent!", fg="green")
 
 
 if __name__ == "__main__":
```

Comment:
Since you are already calling the `cli.main()` function in `[project.scripts]` in your `pyproject.toml` config file you don't need this.